docs/attributes-registry/gen-ai.md

```diff
@@ -0,0 +1,48 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Large Language Model (LLM)
+
+<!-- toc -->
+
+- [Generic LLM Attributes](#generic-llm-attributes)
+  - [Request Attributes](#request-attributes)
+  - [Response Attributes](#response-attributes)
+  - [Event Attributes](#event-attributes)
+
+<!-- tocstop -->
+
+## Generic LLM Attributes
+
+### Request Attributes
+
+<!-- semconv registry.llm(omit_requirement_level,tag=llm-generic-request) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `gen_ai.llm.request.max_tokens` | int | The maximum number of tokens the LLM generates for a request. | `100` |
+| `gen_ai.llm.request.model` | string | The name of the LLM a request is being made to. | `gpt-4` |
+| `gen_ai.llm.request.temperature` | double | The temperature setting for the LLM request. | `0.0` |
+| `gen_ai.llm.request.top_p` | double | The top_p sampling setting for the LLM request. | `1.0` |
+| `gen_ai.llm.system` | string | The name of the LLM foundation model vendor, if applicable. | `openai` |
+<!-- endsemconv -->
+
+### Response Attributes
+
+<!-- semconv registry.llm(omit_requirement_level,tag=llm-generic-response) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `gen_ai.llm.response.finish_reason` | string[] | Array of reasons the model stopped generating tokens, corresponding to each generation received. | `[['stop']]` |
```

Comment:
maybe

```suggestion
| `gen_ai.llm.response.finish_reasons` | string[] | Array of reasons the model stopped generating tokens, corresponding to each generation received. | `['stop']` |
```

?

Reply:
I think there is only one finish_reason? If so, this should be type `string` instead of `string[]`.

Reply:
No, there might be more than one - OpenAI for example returns multiple results and hence there are multiple finish reasons.